Makefile

```diff
@@ -175,7 +175,8 @@ push_python_regressions:
 	gsutil -m cp -r $(TEST_DATA_HOST)/python_regressions $(REGRESSION_DATA_STORAGE_BUCKET)/$(FORTRAN_SERIALIZED_DATA_VERSION)/$(EXPERIMENT)/python_regressions
 
 get_test_data:
-	if [ ! -f "$(TEST_DATA_HOST)/input.nml" ]; then \
+	if [ ! -f "$(TEST_DATA_HOST)/input.nml" ] || [ "$$(gsutil cat $(REGRESSION_DATA_STORAGE_BUCKET)/$(FORTRAN_SERIALIZED_DATA_VERSION)/${EXPERIMENT}/md5sums.txt)" != "$$(cat $(TEST_DATA_HOST)/md5sums.txt)"  ]; then \
```

Comment:
is it possible to break this onto shorter lines and use backslashes?

Reply:
not sure -- Makefile syntax is uber picky, but it can probably be broken into multiple lines